CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,192 @@
+
+* Privileges:
+
+   * Ability to be assigned to Issues and ask for Reviews
+
+Becoming a Community Contributor is the first step towards becoming an
+Organization Member.
+
+## Organization Member
+
+An Organization Member is an established contributor who regularly
+participates in the project. Organization Members have privileges in project
+repositories and as such are expected to act in the interests of the whole
+project. Organization Members can take on
+[additional roles within the project](CONTRIBUTOR-ROLES.md) and can work more
+independently, for example triggering CI.
+
+An Organization Member has all the rights and responsibilities of a Community
+Contributor, plus:
+
+* Responsibilities:
+
+   * Continues to contribute regularly
+   * Work to accomplish the tasks they volunteer to do within the project
+
+* Qualifications:
+
+   * Must have successful contributions to the project, including at least one
+   of the following:
+      * Written and/or reviewed PRs,
+      * Reviewed PRs,
+      * Made contributions that resolved Issues,
+      * Or some equivalent combination or contribution
+   * Must be actively contributing to at least one project area, where the
+   privileges of Organization Member will be beneficial
+
+* Privileges:
+
+   * Can trigger CI
+   * Can leave reviews on PRs
+   * Can recommend other contributors to become Metal3 Github Org Members
+   * Can nominate themselves to [other roles within Metal3](CONTRIBUTOR-ROLS.md)
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open an issue in the [community repo](https://github.com/metal3-io/community)
+1. Assign the issue to a [maintainer](OWNERS).
+
+## Reviewer
+
+Description: A Reviewer has responsibility for specific code, documentation,
+test, or other project areas. They are collectively responsible, with other
+Reviewers, for reviewing all changes to those areas and indicating whether those
+changes are ready to merge. They have a track record of contribution and review
+in the project.
+
+Reviewers are responsible for specific repositories and/or technical areas.
+Most often it is one or more related Git repositories. Reviewers have all the
+rights and responsibilities of an Organization Member.
+
+* Responsibilities:
+
+   * Regularly reviewing Pull Requests against their specific areas of
+    responsibility, including Pull Requests that are assigned to them
+
+   * Helping other contributors become reviewers
+
+* Requirements:
+
+   * Is an Organization Member
+   * Has a track record of constructive and valuable reviews
+   * Has demonstrated knowledge of the specific area, for example by resolving
+   test failures or through contributions
+   * At least 10 Commits in the specific repository.
+   * Is supportive of new and occasional contributors and helps get useful PRs
+   in shape to commit
+
+* Privileges:
+
+   * Has GitHub rights to `lgtm` pull requests in specific directories
```

Comment:
```suggestion
   * Can `/lgtm` Pull Requests in specific directories
```
It is not a GH right per se, maybe just omit the type.

We also make it either Prow-specific (tools dependant, specific), or remove the backticks and just make it LGTM (tool independent choice)